extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -195,11 +197,22 @@ export class AmpAdUIHandler {
       resizeInfo.success = false;
       return Promise.resolve(resizeInfo);
     }
+    const performance = Services.performanceForOrNull(this.baseInstance_.win);
+    const activated =
+      event && event.userActivation && event.userActivation.hasBeenActive;
     return this.baseInstance_.attemptChangeSize(newHeight, newWidth).then(
       () => {
+        if (performance) {
+          // Report false positives.
+          performance.tickDelta('rsfp', activated ? 1 : 1000);
```

Comment:
why 1 & 1000?
for simplicity, shall we just use user error?

Reply:
Just expanding the reporting range. Really any numbers would do. User error however seems wrong and distracting and we don't really have good way of charting them, like we do with latency-style dashboards.